### DISCUSSION TITLE
Is this comfortable with Client-Crediential-Flow

### DISCUSSION THREAD

**Discussion by ClusterH** [gilbarbara/react-spotify-web-playback#119] [Q&A] [answered]
Currently I'm developing react, node.js application by integrating spotify api, I want to play music when users visit home page, and play another music when users visit another pages from any specific album(I have album id) automatically.
Since all music should be play start when users visit website, there are no any Authorization (with spotify account login) part in the website. So, I tried follow Client Crediential Flow in the backend and I got access_token from `https://accounts.spotify.com/api/token` endpoint in backend, but did not specify any kind scope to generate token.
My question is I can use this library to use in my case? If so, where should I put scope definition when I call above api?
Here is [stackoverflow question](https://stackoverflow.com/questions/74917984/spotify-api-integration-to-play-special-album-musics-in-react-app-with-spotify-c) I posted with same question.

**Comment by gilbarbara** [comment, marked as answer]
Hey @ClusterH 

You need an access token with user permissions for the player so that the [Client Credential](https://developer.spotify.com/documentation/general/guides/authorization/) won't work. 
And you can't use your account in several instances simultaneously. Check Spotify's [docs](https://developer.spotify.com/documentation/web-playback-sdk/) for more information.
